app/src/main/kotlin/com/mapbox/vision/teaser/MainActivity.kt

```diff
@@ -82,15 +81,22 @@ class MainActivity : BaseTeaserActivity(), ReplayModeFragment.OnClickModeItemLis
         return true
     }
 
-
     private fun showReplayModeFragment() {
         supportFragmentManager
                 .beginTransaction()
                 .replace(R.id.fragment_container, ReplayModeFragment.newInstance(BASE_SESSION_PATH), ReplayModeFragment.TAG)
                 .addToBackStack(ReplayModeFragment.TAG)
                 .commit()
         hideDashboardView()
+    }
 
+    private fun showRecordingFragment() {
```

Comment:
how about joining with prev method?

Reply:
methods will have different input params in next PRs, IMO better to keep separate